mrf24j.h

```diff
@@ -220,6 +220,8 @@ class Mrf24j
         void set_palna(boolean enabled);
 
         void send16(word dest16, char * data);
+        
+        void send16(word dest16, char * data, byte length);
```

Comment:
You should make the header change in the same commit as the .cpp change.  Each individual commit should contain what is needed for the change.  By splitting the commits in two, you have a commit that doesn't compile properly.

You can use git rebase to squish them back into one commit.